### ISSUE TITLE
What's the point of publishing the source code if it's not licensed under a free software license?

### ISSUE THREAD

**Issue by MichaelAgarkov** [WinampDesktop/winamp#237]
I don't see a point on why they would want to do this.

The current company that owns Winamp seems to have done it to trick people on thinking it's open-sourced, when it's in fact not, just to get publicity out of this.

**Comment by cobrien666** [comment]
Especially when you take into consideration that when you submit any changes, you are assigning all rights including intellectual and copyright to Winamp.

**Comment by nq4t** [comment]
Because they think everyone is so nostaligic for it we'll be their slaves. We'll build thier stuff and get nothing in return and like it. 

What they fail to realize is that no one is going to take them seriously anymore. We will gladly allow the legacy of winamp to die an embarassment than be nostalgic about it.

**Comment by HughF** [comment]
Where else are they supposed to put the code if they want it to be available for reference? At least putting it on GitHub it’s accessible.

**Comment by MichaelAgarkov** [comment]
> Where else are they supposed to put the code if they want it to be available for reference? At least putting it on GitHub it’s accessible. More accessible than a zip file on a random URL.

That's not what GitHub is for. They have their own website where they could publish it with a non-free license. Even just run doxygen on the source and give us the generated htmls, but publishing it on GitHub is not a good idea.

**Comment by HughF** [comment]
> > Where else are they supposed to put the code if they want it to be available for reference? At least putting it on GitHub it’s accessible. More accessible than a zip file on a random URL.
> 
> That's not what GitHub is for. They have their own website where they could publish it with a non-free license. Even just run doxygen on the source and give us the generated htmls, but publishing it on GitHub is not a good idea.

I disagree, this is by far the most appropriate place for this code to live. Not on their own website. We’ve seen countless examples of link rot over the years and any steps that can be taken to mitigate that should be followed imho.

**Comment by MichaelAgarkov** [comment]
> > > Where else are they supposed to put the code if they want it to be available for reference? At least putting it on GitHub it’s accessible. More accessible than a zip file on a random URL.
> > 
> > 
> > That's not what GitHub is for. They have their own website where they could publish it with a non-free license. Even just run doxygen on the source and give us the generated htmls, but publishing it on GitHub is not a good idea.
> 
> I disagree, this is by far the most appropriate place for this code to live. Not on their own website. We’ve seen countless examples of link rot over the years and any steps that can be taken to mitigate that should be followed imho.

0,89$ was credited to your bank account.

**Comment by marc2k3** [comment]
I can see people contributing clear and obvious one line bug fixes and not much else.

Not a single person is going to work on any meaningful new feature that would take time/testing. That would be insanity.

**Comment by MichaelAgarkov** [comment]
> I can see people contributing clear and obvious one line bug fixes and not much else.
> 
> Not a single person is going to work on any meaningful new feature that would take time/testing. That would be insanity.

clueless, hasn't read the EULA

**Comment by HughF** [comment]
> I can see people contributing clear and obvious one line bug fixes and not much else.
> 
> Not a single person is going to work on any meaningful new feature that would take time/testing. That would be insanity.

It’s an interesting historical reference only, no-one will be contributing anything. And that’s fine. The code needs to be preserved somewhere.

**Comment by MichaelAgarkov** [comment]
> The code needs to be preserved somewhere.
Then why not archive the repo or put it on archive.org instead then?

**Comment by HughF** [comment]
> > The code needs to be preserved somewhere.
> 
> Then why not archive the repo or put it on archive.org instead then?

I agree, they should archive the repo. I don’t agree that it should be anywhere else though. This is a great place to archive code. 

They’ve messed up with their ‘license’ and the way they’ve handled the community, but I do applaud them for releasing the code.

**Comment by LightYagami28** [comment]
What if you contact the individual companies whose copyright has been infringed to request a DMCA?

**Comment by MichaelAgarkov** [comment]
This is bad PR if anything.

**Comment by hoehermann** [comment]
I think the main issue here is that people keep confusing "open source" with "free software" (as in "free beer").
